R/truncate_linelist.R

```diff
@@ -1,55 +1,39 @@
 #' Adjust or subset a line list to account for right truncation
 #'
 #' @description
-#' Adjust or subset the line list `<data.frame>` by either changing dates that
-#' are within the right-truncation window to `NA` or removing cases that have
-#' occurred within a delay period. This occurs to recent cases or outcomes
-#' that are not yet recorded in the line list and will be revised upwards. See
-#' `truncation_event` argument documentation for different types of truncation.
+#' Adjust or subset the line list `<data.frame>` by removing cases that
+#' have not been reported by the truncation time and setting hospitalisation
+#' admission or outcome dates that are after the truncation point to `NA`.
+#'
+#' This is to replicate real-time outbreak data where recent cases or outcomes
+#' are not yet observed or reported (right truncation).
 #'
 #' @details
-#' The truncation window is sampled for each individual and is specified by a
-#' (random) number generating function supplied to the `delay`
-#' argument. Therefore, if this function has variability
-#' (e.g. a probability distribution) it will not be a fixed time window for
-#' all individuals. It is possible that someone with a more recent date of
-#' onset is kept but someone with a less recent date of onset is removed due
-#' to the variability in the reporting delays. See examples for variable or
-#' fixed `delay` functions.
+#' The point at which the line list is right-truncated is the same for
```

Comment:
```suggestion
#' The date on which the line list is truncated is the same for
```

It's the delay distribution from onset to report that's right truncated, not the linelist itself.

As this is the first time that truncation is mentioned perhaps introduce this above, e.g. say "Truncate the line list `<data.frame>` by removing cases that have not been reported by the truncation time" there, or change the wording here?

Reply:
I would consider more generally replacing truncation point with truncation day throughout as I think it's clearer.